.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,7 +41,7 @@ Thank you!
 
 ### 📝 Change Log
 
-> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
+> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)! Track your changes, like a cat tracks a laser pointer.
```

Comment:
"Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracking a laser pointer." is better wording.